src/constants/index.js

```diff
@@ -350,3 +350,8 @@ export const EVENT_SORT_OPTIONS = [
 export const SORT_DIRECTION = { up: 'up', down: 'down' };
 
 export const DEFAULT_EVENT_SORT = [SORT_DIRECTION.down, EVENT_SORT_OPTIONS[0]];
+
+export const FORM_FIELDS_TYPES = {
```

Comment:
Is this going to be used anywhere int he app? If not, this is probably a too broad place to put it. We could have a constants file specifically under the `SchemaForm` component.